packages/tools/devtools/devtools-browser-extension/e2e-tests/chromebrowserdevtool.test.ts

```diff
@@ -0,0 +1,42 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { globals } from "../jest.config";
+import { retryWithEventualValue } from "@fluidframework/test-utils";
+// import { exec } from "child_process";
+
+describe("chrome browser...", () => {
```

Comment:
It doesn't look like this verifies anything about the devtools panel. Can we add validation that the devtools UI is on the screen as well?

Reply:
I'm planning to include extension test cases in next pr to sperate the jest configuration and tests so if it has problem we don't need to revoke everything. This test case is to verify jest configuration is correct.

Reply:
I guess my concern is that if we haven't verified that this approach will let us test the extension, then we probably don't want all of this infrastructure. Have you been able to verify that we can perform interactive tests on the extension UI?

Reply:
Yes. we can find the extension id in test and use `page.goto(chrome-extension://${extensionId}/devtools.html#/)` to interactive with extension UI.